react/features/video-quality/components/VideoQualityDialog.js

```diff
@@ -0,0 +1,302 @@
+import InlineMessage from '@atlaskit/inline-message';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import {
+    setAudioOnly,
+    setReceiveVideoQuality,
+    VIDEO_QUALITY_LEVELS
+} from '../../base/conference';
+
+import { translate } from '../../base/i18n';
+
+const {
+    HIGH,
+    MEDIUM,
+    LOW
+} = VIDEO_QUALITY_LEVELS;
+
+/**
+ * Implements a React {@link Component} which displays a dialog with a slider
+ * for selecting a new receive video quality.
+ *
+ * @extends Component
+ */
+class VideoQualityDialog extends Component {
+    /**
+     * {@code VideoQualityDialog}'s property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Whether or not the conference is in audio only mode.
+         */
+        _audioOnly: React.PropTypes.bool,
+
+        /**
+         * Whether or not the conference is in peer to peer mode.
+         */
+        _p2p: React.PropTypes.bool,
```

Comment:
It is a boolean, not an object. I used the convention from JitsiConference where its attribute p2p is a boolean. I can rename it something like isP2P or whatever you think is more clear.